src/helpers.test.js

```diff
@@ -375,6 +378,45 @@ describe('getPrettierEslintOptions', () => {
   });
 });
 
+describe('getEditorConfigOptions', () => {
+  test('parse editorconfig by filename', () => {
+    getEditorConfigOptions('filename');
+    expect(editorconfig.parseSync).toHaveBeenCalledWith('filename');
+  });
+  test('maps editorconfig options', () => {
+    editorconfig.parseSync.mockImplementation(() => ({
+      indent_size: 4,
+      end_of_line: 'lf',
+      charset: 'utf-8',
+      trim_trailing_whitespace: true,
+      insert_final_newline: true,
+      tab_width: 2,
+      max_line_length: 100,
+    }));
+    const expected = {
+      tabWidth: 2,
+      printWidth: 100,
+    };
+    expect(getEditorConfigOptions('')).toEqual(expected);
+  });
+
+  test('returns boolean for useTabs option', () => {
+    const setIndentStyle = (style) => {
+      editorconfig.parseSync.mockImplementation(() => ({
+        indent_style: style,
+      }));
+    };
+    const getUseTabsConfig = () => {
+      const options = getEditorConfigOptions('');
+      return options ? options.useTabs : null;
+    };
+    setIndentStyle('tab');
+    expect(getUseTabsConfig()).toEqual(true);
+    setIndentStyle('space');
+    expect(getUseTabsConfig()).toEqual(false);
+  });
```

Comment:
Let's just use two tests here so that it's a bit clearer what's going on and we don't have to write a bunch of custom code. Something like:

```js
  test('returns true if editor config indent_style is set to "tab"', () => {
    editorconfig.parseSync.mockImplementation(() => ({
      indent_style: 'tab',
    }));

    const actual = getEditorConfigOptions('fakeFilepath.js').useTabs;

    expect(actual).toBe(true);
  });
  
  test('returns false if editor config indent_style is set to "spaces"', () => {
    editorconfig.parseSync.mockImplementation(() => ({
      indent_style: 'spaces',
    }));

    const actual = getEditorConfigOptions('fakeFilepath.js').useTabs;

    expect(actual).toBe(false);
  });
```